astroquery/vo_conesearch/conesearch.py

```diff
@@ -78,21 +74,33 @@ class AsyncConeSearch(AsyncBase):
     If no ``timeout`` keyword given, it waits until completion:
 
     >>> async_result = async_search.get(timeout=30)
-    >>> cone_arr = async_result.array.data
-    >>> cone_arr.size
-    36184
+    >>> len(async_result)
+    74271
 
     """
     def __init__(self, *args, **kwargs):
         super(AsyncConeSearch, self).__init__(conesearch, *args, **kwargs)
 
 
-def conesearch(center, radius, verb=1, catalog_db=None, pedantic=None,
-               verbose=True, cache=True, timeout=None, query_all=False):
+def conesearch(center, radius, verb=1, catalog_db=None,
+               verbose=True, cache=True, query_all=False,
+               return_astropy_table=True, use_names_over_ids=False):
     """
     Perform Cone Search and returns the result of the
     first successful query.
 
+    .. note::
+
+        Use ``astroquery.vo_conesearch.conf.pedantic`` to control
```

Comment:
should we rather deprecate this first? you know better the userbase to judge it.

Reply:
User base is not wide enough to justify the maintenance overhead.